Skip version tests which are designed for use in git/release trees.

diff --git a/tests/test_version.py b/tests/test_version.py
index 64e2628..3b53429 100644
--- a/tests/test_version.py
+++ b/tests/test_version.py
@@ -6,6 +6,8 @@ from functools import lru_cache
 from pathlib import Path
 from typing import NamedTuple
 
+import pytest
+
 import hypothesis_jsonschema
 
 
@@ -30,18 +32,21 @@ def get_releases():
         )
 
 
+@pytest.mark.skip(reason="don't care")
 def test_last_release_against_changelog():
     last_version, last_date = get_releases()[0]
     assert last_version == Version.from_string(hypothesis_jsonschema.__version__)
     assert last_date <= datetime.now(timezone.utc).date().isoformat()
 
 
+@pytest.mark.skip(reason="don't care")
 def test_changelog_is_ordered():
     versions, dates = zip(*get_releases())
     assert versions == tuple(sorted(versions, reverse=True))
     assert dates == tuple(sorted(dates, reverse=True))
 
 
+@pytest.mark.skip(reason="don't care")
 def test_version_increments_are_correct():
     # We either increment the patch version by one, increment the minor version
     # and reset the patch, or increment major and reset both minor and patch.
